src/Command/CreateBuildingCommand.php

```diff
@@ -47,6 +50,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->entityManager->persist($building);
         $this->entityManager->flush();
 
+        $this->logger->info(
+            'Successfully created building: ',
```

Comment:
The colon at the end of the message has no sense, the log will be processed and the context most probably wont be just printed after the message.